tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -198,6 +228,10 @@ install_cached_tarball_component() {
 	# "tarball1_name:tarball1_path tarball2_name:tarball2_path ... tarballN_name:tarballN_path"
 	local extra_tarballs="${6:-}"
 
+	if [ "${component}" = "shim-v2" ]; then
```

Comment:
So just to check I understand this code - `install_cached_shim_v2_tarball_get_root_hash` gets the root_hash from the github artifact rootfs dependency, then `install_cached_shim_v2_tarball_compare_root_hashes` is used to compare that to the cached version of the shimv2 to tell if it's updated and therefore a cache failures, or it's fine so we can proceed?

Reply:
Exactly!

The reason we cannot do those 2 at the same point is because:
* If, for some other reason, we need to rebuild the shim, we need to guarantee that the root_hash.txt file exists, thus getting it as early as possible
* And where we get it, the ORAS image is not yet pulled, thus we don't have something to compare the root_hash.txt generated as part of the build process to.